CHANGELOG.md

```diff
@@ -4,6 +4,162 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.30.0] - 2023-09-14
+
+## 🚀 Features
+
+### Rhai Support at the `router_service` ([Issue #2278](https://github.com/apollographql/router/issues/2278))
+
+It is now possible to interact with requests and responses at the `router_service` level [using Rhai-based customizations](https://www.apollographql.com/docs/router/customizations/rhai/). The functionality is very similar to that provided for interacting with existing services, for example `supergraph_service`. For instance, you may "map" requests and responses as follows:
```

Comment:
```suggestion
It is now possible to intercept requests and responses at the `router_service` level [using Rhai-based customizations](https://www.apollographql.com/docs/router/customizations/rhai/). The functionality mirrors existing functionality router services, for example `supergraph_service`. For example, "maping" requests and responses as follows:
```

Reply:
I'll resolve this as I don't think _intercept_ is actually the right terminology to describe how these layers work.  The other changes you suggest 
are almost identical.  (Maybe regional? e.g., for instance vs for example 😄 )